galasa-parent/dev.galasa.framework/src/main/java/dev/galasa/framework/BackupCPS.java

```diff
@@ -0,0 +1,178 @@
+/*
+ * Licensed Materials - Property of IBM
+ * 
+ * (c) Copyright IBM Corp. 2020.
+ */
+
+package dev.galasa.framework;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.StandardOpenOption;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+import java.util.TreeMap;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
+import org.osgi.service.component.annotations.Component;
+
+import dev.galasa.framework.FrameworkInitialisation;
+import dev.galasa.framework.spi.FrameworkException;
+import dev.galasa.framework.spi.IConfigurationPropertyStoreService;
+import dev.galasa.framework.spi.IFramework;
+
+@Component(service = { BackupCPS.class })
+public class BackupCPS {
+    
+    private Log             logger  =  LogFactory.getLog(this.getClass());
+    
+    private StringBuilder   sb;
+    private Path            path;
+    
+    IFramework framework;
```

Comment:
should be private,    think the default is protected

Reply:
Done 👍